src/copy.rs

```diff
@@ -0,0 +1,89 @@
+use std::fmt;
+
+use serde::{de, ser};
+
+/// Defines a supported clipboard copy command
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct CopyCmd {
+    /// Command to perform a copy to the clipboard
+    pub(crate) cmd: &'static str,
+    /// Arguments to pass to the copy command
+    pub(crate) args: Vec<&'static str>,
+}
+
+/// Defines the supported clipboard copy command.
+/// A `Vec<CopyCmd>` is returned according to the conditional compilation on the detected OS.
+/// The first item declared is considered the default copy command.
+fn get_supported_copy_cmds() -> Vec<CopyCmd> {
+    #[cfg(target_os = "linux")]
```

Comment:
Would be good to add a case where it doesn't match any of these OSes and returns an empty list